src/app/shared/mocks/in-memory-vendor-data.service.ts

```diff
@@ -67,11 +67,10 @@ export class InMemoryVendorDataService implements InMemoryDbService {
     return { vendors };
   }
 
-  // Overrides the genId method to ensure that a hero always has an id.
```

Comment:
sorry I forgot this line with '**vendor** always has an id'
``` // Overrides the genId method to ensure that a vendor always has an id. ```
**vendor** always has an id

it's not very important, but ...